cmd/distribution_account_cmd_service.go

```diff
@@ -113,21 +112,22 @@ func (s *DistributionAccountService) createNewStellarAccountFromAccount(ctx cont
 }
 
 func (s *DistributionAccountService) buildCreateAccountOperation(ctx context.Context, newAccount schema.TransactionAccount, numberOfTrustlines int) txnbuild.Operation {
-	baseReserveFee := 0.5
-	minimumFundingAmount := (2 * baseReserveFee) + (float64(numberOfTrustlines) * baseReserveFee) // 2 base reserves to exist on the ledger + 1 base reserve per trustline
-	fundingAmount := math.Max(float64(s.nativeAssetBootstrapAmount), minimumFundingAmount)
+	baseReserveFee := decimal.RequireFromString("0.5")
```

Comment:
```suggestion
baseReserveFee := decimal.NewFromFloat(0.5)
```
This is probably better than using `RequireFromString` which can panic.